pkgs/development/misc/haskell/ap-normalize/default.nix

```diff
@@ -0,0 +1,17 @@
+{ mkDerivation, base, fetchgit, inspection-testing, lib
+, transformers
+}:
+mkDerivation {
+  pname = "ap-normalize";
+  version = "0.1.0.1";
+  src = fetchgit {
```

Comment:
Lets use fetchFromGitLab.

Reply:
As said [above](https://github.com/NixOS/nixpkgs/pull/130946#discussion_r674075476), regenerating the file using `cabal2nix` will cause it to fetch the sdist tarball from hackage, which is preferrable in every way.